ja-odur/puzzled
# #167557224 implement solve sudoku puzzle

**What does this PR do?**
* Implement solve sudoku puzzle on the frontend

**Description of Task to be completed?**
* Implement solve sudoku puzzle on the frontend

**How should this be manually tested?**
* Pull and checkout to this branch locally by running `git fetch origin ft-solve-sudoku-pazzle-167557224 && ft-solve-sudoku-pazzle-167557224`
* Start  the virtual environment `pipenv shell`
* install python requirements `pipenv install`
* install node requirements `npm install`
* migrate the database `python manage.py migrate`
* run builds `npm run build`
* start the django server `python manage.py runserver`
* Hit the endpoint `http://127.0.0.1:8000/` and click on the first sudoku card, you see a grid.

**Any background context you want to provide?**
* N/A

**What are the relevant pivotal tracker stories?**
* [167557224](https://www.pivotaltracker.com/story/show/167557224)
